src/mesh/NodeDB.cpp

```diff
@@ -137,6 +143,31 @@ void NodeDB::installDefaultRadioConfig()
 
 }
 
+void NodeDB::installDefaultCannedMessagePluginPart1()
```

Comment:
These should go into the plugin directly, not the NodeDB. They're outside of the function of the NodeDB.

Reply:
Since these CannedMessage settings are more like user preferences, they need to be along side the RadioConfig (which has UserPrefs). They need to be saved/loaded. So, I'm not sure how I might push these values out of NodeDB. Where might those values be moved to?